platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/TaskChatInSpaceRoomTestIT.java

```diff
@@ -0,0 +1,166 @@
+import org.junit.jupiter.api.Test;
+
+import static com.codeborne.selenide.Selectors.*;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.switchTo;
+import static org.exoplatform.platform.qa.ui.core.PLFData.DATA_NAME_USER1;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomNumber;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomString;
+import static org.exoplatform.platform.qa.ui.selenium.locator.chat.ChatLocator.*;
+import static org.exoplatform.platform.qa.ui.selenium.locator.taskmanagement.TaskManagementLocator.*;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+import static org.exoplatform.platform.qa.ui.selenium.testbase.LocatorTestBase.ELEMENT_SKIP_BUTTON;
+
+@Tag("task")
+@Tag("sniff")
+public class TaskChatInSpaceRoomTestIT extends Base {
+    HomePagePlatform homePagePlatform;
+    SpaceManagement spaceManagement;
+    SpaceHomePage spaceHomePage;
+    TasksManagement tasksManagement;
+    ManageLogInOut manageLogInOut;
+    LabelsManagement labelsManagement;
+    ProjectsManagement projectsManagement;
+    RoomManagement roomManagement;
+    NavigationToolbar navigationToolbar;
+    UserAddManagement userAddManagement;
+    SpaceSettingManagement spaceSettingManagement;
+    AddUsers addUsers;
+
+    @BeforeEach
+    public void setupBeforeMethod() {
+        info("Start setUpBeforeMethod");
+
+        homePagePlatform = new HomePagePlatform(this);
+        spaceHomePage = new SpaceHomePage(this);
+        spaceManagement = new SpaceManagement(this);
+        tasksManagement = new TasksManagement(this);
+        labelsManagement = new LabelsManagement(this);
+        projectsManagement = new ProjectsManagement(this);
+        roomManagement = new RoomManagement(this);
+        navigationToolbar = new NavigationToolbar(this);
+        userAddManagement = new UserAddManagement(this);
+        spaceSettingManagement = new SpaceSettingManagement(this);
+        addUsers = new AddUsers(this);
+        manageLogInOut = new ManageLogInOut(this);
+        manageLogInOut.signInCas(PLFData.DATA_USER1, "gtngtn");
+        if ($(ELEMENT_SKIP_BUTTON).is(Condition.exist)) {
+            $(ELEMENT_SKIP_BUTTON).click();
+        }
+    }
+
+    @Test
+    public void test01_TaskInSpaceRoomWithNoAssignee() {
+
+        String space = "space" + getRandomNumber();
+        String task = "task" + getRandomNumber();
+
+        homePagePlatform.goToMySpaces();
+        spaceManagement.addNewSpaceSimple(space, space, 6000);
+        homePagePlatform.goToChat();
+        switchTo().window(1);
+        ELEMENT_CONTACT_LIST.find(byText(space)).click();
+        ELEMENT_COLLABORATION_ACTIONS.click();
+        ELEMENT_CHAT_CREATE_TASK.click();
+        ELEMENT_CHAT_TASK_NAME.setValue(task);
+        ELEMENT_CHAT_POST_TASK_BUTTON.click();
+        ELEMENT_CONTAINER_LIST_MESSAGES.find(byLinkText(task)).shouldBe(Condition.visible);
```

Comment:
you can create a function createTaskInChat() to reuse it. (same for other tests)

Reply:
done